source/Engine.cpp

```diff
@@ -1394,6 +1395,17 @@ void Engine::EnterSystem()
 		Messages::Add(GameData::HelpMessage("basics 1"), Messages::Importance::High);
 		Messages::Add(GameData::HelpMessage("basics 2"), Messages::Importance::High);
 	}
+
+	// Create the planet labels.
+	labels.clear();
+	if(system)
+		for(const StellarObject &object : system->Objects())
+			if(object.HasSprite() && object.HasValidPlanet() && object.GetPlanet()->IsAccessible(flagship))
+				labels.emplace_back(labels, *system, object);
```

Comment:
...Is it messed up? I presumed it showed up because that was added to `master` at that point in the file between when I branched and when I rebased...

Reply:
That code was moved somewhere else, it should be removed from here. GitHub doesn't show commits from the master branch.

Reply:
OK, thanks for catching it, then. I'll take it out.